### PR TITLE
Update hardware_setup.md

### PR DESCRIPTION
**Pull request by ToniRV**

### CODE REVIEW

File: docs/hardware_setup.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ It is important to remember that when launching the VIO, the camera should be st
 
 3. Calibrate camera intrinsics and extrinsics using [Kalibr](https://github.com/ethz-asl/kalibr), recommended model is: ```pinhole-equi``` [(see OpenCV documentation)](https://docs.opencv.org/3.3.1/db/d58/group__calib3d__fisheye.html)
 
-4. Create configuration files for KimeraVIO ROS wrapper using [Kalibr2KimeraVIO-pinhole-equi](https://github.mit.edu/SPARK/VIO/blob/feature/parallelization/jpl/kalibr/kalibr2sparkvio_stereo_pinhole-equi.py) or [[Kalibr2KimeraVIO-pinhole-radtan](https://github.mit.edu/SPARK/VIO/blob/feature/parallelization/jpl/kalibr/kalibr2sparkvio_stereo_pinhole-equi.py)]
+4. Create configuration files for KimeraVIO ROS wrapper using [Kalibr2KimeraVIO](https://github.com/MIT-SPARK/Kimera-VIO/kalibr2kimeravio.py)
 
-5. Create/adapt your own specific launch file, similar to [example MyntEye S](https://github.mit.edu/SPARK/spark_vio_ros/blob/jpl/launch/spark_vio_ros_mynteye.launch)
+5. Create/adapt your own specific launch file, similar to [example MyntEye S](https://github.com/MIT-SPARK/Kimera-VIO-ROS/kimera_ros_mynteye.launch)
```
Comment (msadowski) [inline docs/hardware_setup.md]: This address returns 404, Should probably point to: https://github.com/MIT-SPARK/Kimera-VIO-ROS/blob/master/launch/kimera_ros_mynteye.launch ?

File: docs/hardware_setup.md [hunk 2]
```diff
@@ -49,9 +49,9 @@ It is important to remember that when launching the VIO, the camera should be st
 
 3. Calibrate camera intrinsics and extrinsics using [Kalibr](https://github.com/ethz-asl/kalibr), recommended model is: ```pinhole-equi``` [(see OpenCV documentation)](https://docs.opencv.org/3.3.1/db/d58/group__calib3d__fisheye.html)
 
-4. Create configuration files for KimeraVIO ROS wrapper using [Kalibr2KimeraVIO-pinhole-equi](https://github.mit.edu/SPARK/VIO/blob/feature/parallelization/jpl/kalibr/kalibr2sparkvio_stereo_pinhole-equi.py) or [[Kalibr2KimeraVIO-pinhole-radtan](https://github.mit.edu/SPARK/VIO/blob/feature/parallelization/jpl/kalibr/kalibr2sparkvio_stereo_pinhole-equi.py)]
+4. Create configuration files for KimeraVIO ROS wrapper using [Kalibr2KimeraVIO](https://github.com/MIT-SPARK/Kimera-VIO/kalibr2kimeravio.py)
```
Comment (msadowski) [inline docs/hardware_setup.md]: This one also gives me 404 (but I can see it here: https://github.com/MIT-SPARK/Kimera-VIO/blob/fix/names_kalibr_files/config2kimeravio.py)

File: docs/hardware_setup.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ The infrared cameras offer the option to run the KimeraVIO stereo version on mon
 
 7. Create configuration files for KimeraVIO ROS wrapper using [Kalibr2KimeraVIO-pinhole-radtan](https://github.com/MIT-SPARK/KimeraVIO/blob/feature/parallelization/jpl/kalibr/kalibr2sparkvio_stereo_pinhole-radtan.py)
 
-8. Create/adapt your own specific launch file, similar to [example RealSense IR](https://github.mit.edu/SPARK/spark_vio_ros/blob/jpl/launch/spark_vio_ros_realsense_IR.launch)
+8. Create/adapt your own specific launch file, similar to [example RealSense IR](https://github.com/MIT-SPARK/Kimera-VIO-ROS/kimera_ros_realsense_IR.launch)
```
Comment (msadowski) [inline docs/hardware_setup.md]: This returns "Not Found" for me. Probably should point to https://github.com/MIT-SPARK/Kimera-VIO-ROS/blob/master/launch/kimera_ros_realsense_IR.launch

File: docs/hardware_setup.md (hunk 1 above)
Comment (ToniRV) [reply docs/hardware_setup.md]: Yes thanks! Updating now.

**Pull request merged**